htsget_server/htsget_operations.py

```diff
@@ -171,50 +162,48 @@ def get_variants_data(id_, reference_name=None, format_="VCF", start=None, end=N
     return None, auth_code
 
 
-@app.route('/variants/<path:id_>/verify')
-def verify_variants_analysis_drs_object(id_):
-    try:
-        auth_code = authz.is_authed(escape(id_), connexion.request)
-        if auth_code == 200:
-            _verify_analysis_drs_object(id_)
-        else:
-            return {"message": "User is not authorized to verify variants"}, 403
-    except Exception as e:
-        return {"result": False, "message": str(e)}, 200
-    return {"result": True}, 200
-
-
-@app.route('/variants/<path:id_>/index')
-def index_variants(id_=None, force=False, do_not_index=False, genome='hg38'):
+@app.route('/<path:id_>/index')
+def index_analysis(id_=None, force=False, do_not_index=False, genome='hg38'):
```

Comment:
Mildly confusing that there's a function called `index_analysis` that you can specify `do_not_index`` -- I don't think I fully understand what it's doing if that flag is true. Does it just return the indexing status?

Reply:
Honestly I think this was a hack I put in to make some integration tests work...I'm not sure if it's necessary, but at this point I'm scared to take it out. Although if I'm changing the paths anyway, this might be a good time to take it out.